setup.py

```diff
@@ -19,34 +19,38 @@
 # License along with this program.  If not, see
 # <http://www.gnu.org/licenses/>.
 
-"""
-Created on Thu Nov 24 15:18:15 2016
-Setup skript for installing primo and it's dependencies.
-@author: jpoeppel
-"""
-
-from setuptools import setup
+import setuptools
 import sys
 
 if sys.argv[-1] == 'setup.py':
     print("To install, run 'python setup.py install'")
     print()
 
-version = 0.9
+version = 0.91
 
-setup(name="primo2",
-      version=version,
-      description="(Partial) reimplementation of PRobabilistic Inference MOdules",
-      long_description="This project is a (partial) reimplementation of the original " \
-                       "probabilistic inference modules which can be found at " \
-                       "https://github.com/hbuschme/PRIMO. This reimplementation "\
-                       "follows the same general idea, but restructured and unified the "\
-                       "underlying datatypes to allow a more concise API and more efficient "\
-                       "manipulation, e.g. by the inference algorithm. In turn the inference "\
-                       "algorithms have been rewritten and partly extended. For most if not "\
-                       "all use cases this implementation should be easier to use and more "\
-                       "performant than the original.",
-      author="Jan Pöppel",
-      author_email="jpoeppel@techfak.uni-bielefeld.de",
-      packages = ["primo2", "primo2.inference"],
-      install_requires=["numpy", "networkx", "lxml"])
+setuptools.setup(
+    name="primo2",
+    version=version,
+    description="(Partial) reimplementation of PRobabilistic Inference MOdules",
+    long_description="This project is a (partial) reimplementation of the original " \
+        "probabilistic inference modules which can be found at " \
+        "https://github.com/hbuschme/PRIMO. This reimplementation " \
+        "follows the same general idea, but restructured and unified the " \
+        "underlying datatypes to allow a more concise API and more efficient " \
+        "manipulation, e.g. by the inference algorithm. In turn the inference " \
+        "algorithms have been rewritten and partly extended. For most if not " \
+        "all use cases this implementation should be easier to use and more " \
+        "performant than the original.",
+    author="Jan Pöppel",
```

Comment:
Well, there is the AUTHORS file. We could think about replacing the fields `author` and `author_email` with `maintainer` and `maintainer_email` though, see the [PEP 345](http://legacy.python.org/dev/peps/pep-0345/#maintainer-optional).

We should also think about adding additional metadata attributes listed in PEP 345, e.g., license information and Python compatibility